icc-x-api/icc-crypto-x-api.ts

```diff
@@ -602,6 +604,64 @@ export class IccCryptoXApi {
     )
   }
 
+  saveKeychainInBrowserLocalStorageAsBase64(id: string, keyChainB64: string) {
+    localStorage.setItem(this.keychainLocalStoreIdPrefix + id, keyChainB64)
+  }
+
+  saveKeyChainInHCPFromLocalStorage(hcpId: string): Promise<HealthcarePartyDto> {
+    return this.hcpartyBaseApi
+      .getHealthcareParty(hcpId)
+      .then(hcp => {
+        const crt = this.getKeychainInBrowserLocalStorageAsBase64(hcp.id!!)
```

Comment:
Are you sure at that a certificat is really in the storage at that point ?
=> Check before get HCP ?

Reply:
it seems ok based on the code of Medispring, but the process should be more robust... 
=> check that `this.getKeychainInBrowserLocalStorageAsBase64(hcp.id!!)` return something before get the HCP